test/compound_test_identifier.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from ersilia.utils.identifiers.compound import CompoundIdentifier
+
+@pytest.fixture
+def compound_identifier():
+    return CompoundIdentifier()
+
+def test_is_key_header_positive(compound_identifier):
```

Comment:
I would also recommend looking into [parameterization](https://docs.pytest.org/en/stable/how-to/parametrize.html) in pytest fixtures. @musasizivictoria has done a wonderful job to start us off, you could work together!

Reply:
Sure.